sdk/iot/iot-modelsrepository/test/browser/browserTest.spec.ts

```diff
@@ -6,20 +6,20 @@ import { ModelsRepositoryClient } from "../../src";
 
 describe("resolver -  browser", () => {
   describe("single resolution (no pseudo-parsing)", () => {
-    it.only("integration works in browser", function(done) {
+    it("integration works in browser", function(done) {
```

Comment:
would it make sense to use async functions here so you don't have to worry about calling `done` and can use `await` instead of `then`?